ocrd_tesserocr/ocrd-tool.json

```diff
@@ -172,6 +172,12 @@
           "default": {},
           "description": "Dictionary of additional Tesseract runtime variables (cf. tesseract --print-parameters), string values."
         },
+        "oem": {
```

Comment:
We could add that to the version string added as metadata to the PAGE-XML, so it would be something like `0.11.0 (tesseract 5.0.0-alpha-20201231 - TESSERACT_LSTM_COMBINED)`.

Reply:
yes, but it's already in the `parameter` metadata.

Reply:
True.